assigner/commands/assign.py

```diff
@@ -42,7 +44,14 @@ def assign(conf, args):
         )
         base = BaseRepo(host, namespace, hw_name, token)
         if not dry_run:
-            base.clone_to(tmpdirname, branch)
+            try:
+                base.clone_to(tmpdirname, branch)
+            except RepoError as e:
+                logging.error("Could not clone base repo (have you pushed at least one commit to it?)")
+                logging.debug(e)
+                mkdir(tmpdirname) # Prevent tempfile from complaining about the directory not existing
```

Comment:
When it exits the context manager it tries to delete this directory and without this line it's upset?

Reply:
Yeah. I suppose I could try to catch the exception and squash it.

Reply:
Aha, the problem is actually that the call to GitPython's `clone_from` deletes the directory and then fails to re-create it. 

Why are computers